types/react-event-listener/ts3.1/index.d.ts

```diff
@@ -0,0 +1,220 @@
+import React = require("react");
+
+export interface EventOptions extends Pick<AddEventListenerOptions, 'capture' | 'passive'> { }
+
+export function withOptions<T, TThis = any>(handler: (this: TThis, ev: T) => any, options: EventOptions): (this: TThis, ev: T) => any;
+
+export type EventListenerThisType<T extends EventTarget | WindowEventTargets> = T extends keyof Window ? Window[T] : T;
+
+export type OnErrorEventHandlerArgs = OnErrorEventHandlerNonNull extends (...args: infer A) => any ? A : [];
```

Comment:
The base definition uses `ErrorEventHandler`, which got deleted in `ts@next`. `OnErrorEventHandlerNonNull` has the exact same definition but is only available since `ts@3.1`.